fetch-client/fetched.ts

```diff
@@ -258,9 +257,10 @@ class ResourceFetched implements IResourceInSpace {
   }
 
   async delete(options: {
-    signer?: ISigner
+    signer?: ISigner,
+    headers?: Record<string, string>
```

Comment:
it might be easier to keep this type in sync with the other methods if it is `options: ISignedRequestOptions` but afaict they are equivalent